src/global/i18n.ts

```diff
@@ -64,6 +66,13 @@ export default (function() {
             return untranslated;
           }
         }
+        if (options) {
+          for ( const key of Object.keys(options)){
+            console.log(key);
```

Comment:
accidentally left this console statements in?

Reply:
Remove or convert to debugs if this is useful.

Reply:
yeah I did. They've been removed now